src/test/cli/rbd/help.t

```diff
@@ -293,7 +294,9 @@ Skip test on FreeBSD as it generates different output there.
     --journal-splay-width arg number of active journal objects
     --journal-object-size arg size of journal objects
     --journal-pool arg        pool for journal objects
+    --thick                   thick image provisioned
```

Comment:
Nit: can we change the optional to ```--thick-provision``` w/ a description along the lines of "fully allocate image"